### DISCUSSION TITLE
Pairing worked perfectly once. Now "Not possible to add the accessory" Operation aborted.

### DISCUSSION THREAD

**Discussion by Sill1000** [HomeSpan/HomeSpan#505] [Q&A]
Hello,

Thanks for this nice library. I started with exemple 5 and worked perfectly! 

I would like to test another example but I never could pair it again :-( After entry the pairing code: "Not possible to add the accessory" Operation aborted. 

I blocked my router on the 2.4GHz. With discovery _hap_tcp is here.  IP adress is present. Ping is ok. 
ard-esp32 = 2.0.6
board esp32
hspn 1.7.1
what to do ? 
Of course I tried with the monitor: factory reset, erase all. No change ! :-( What to do ?

Thanks !

**Comment by HomeSpan** [comment]
What do you mean by blocking your router 2.4GHz?  The ESP32 only operates at 2.4 GHz.

**Comment by Sill1000** [comment]
According to the thread #273. Now the router is using only 2.4GHz

**Comment by HomeSpan** [comment]
Are you using the same exact sketch?  In other words, has anything changed, or did you simply unpair the device from HomeKit, erase all saved values, and then tried to re-pair the device (no changes to router, WiFI, etc.).

**Reply by Sill1000** [reply]
I wanted to test another sketch. Afterwards I deleted the device from Homekit.
Do we need to do a CLI U command before downloading a new sketch?

**Reply by HomeSpan** [reply]
Type 'H' into the Serial Monitor.  This will cause HomeSpan to refresh it's Device ID and permit clean pairing.
